From 320fda7867428406771adfd3c4afa91a33fd6845 Mon Sep 17 00:00:00 2001
From: Patrick Griffis <tingping@tingping.se>
Date: Mon, 3 Oct 2016 12:51:23 -0400
Subject: [PATCH 03/10] gnome.generate_gir(): Also include current build dir

Continuation of 084b854ce057ee6d954c24e58321caa92f542bc5

(cherry picked from commit 94b7b59546cfbc311336ec3700393cfff3d4a840)
---
 mesonbuild/modules/gnome.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/mesonbuild/modules/gnome.py b/mesonbuild/modules/gnome.py
index 81cc462..be111ea 100644
--- a/mesonbuild/modules/gnome.py
+++ b/mesonbuild/modules/gnome.py
@@ -225,7 +225,8 @@ class GnomeModule:
 
         extra_args = mesonlib.stringlistify(kwargs.pop('extra_args', []))
         scan_command += extra_args
-        scan_command += ['-I' + os.path.join(state.environment.get_source_dir(), state.subdir)]
+        scan_command += ['-I' + os.path.join(state.environment.get_source_dir(), state.subdir),
+                         '-I' + os.path.join(state.environment.get_build_dir(), state.subdir)]
         scan_command += self.get_include_args(state, girtarget.get_include_dirs())
 
         if 'link_with' in kwargs:
-- 
2.10.1

